templates/pkg/resource/references.go.tpl

```diff
@@ -18,10 +18,28 @@ import (
 	ackerr "github.com/aws-controllers-k8s/runtime/pkg/errors"
 {{ end -}}
 	acktypes "github.com/aws-controllers-k8s/runtime/pkg/types"
-
```

Comment:
keep this separate newline please. it demarcates third-part imports (which the ACK runtime is) from "local" imports (which the specific controller is)